### ISSUE TITLE
[Docs] The bottom line of the Instructions docs page is a bit confusing.

### ISSUE THREAD

**Issue by wujekbogdan** [fuse-box/fuse-box#1071]
The last line of the [Instalation](https://fuse-box.org/page/installation) page might be confusing. 
The last line says:
``` javascript
import { FuseBox } from "fuse-box/es6"
```

... I know that before this line there's an explanation that this should be used only in for node.js < 8.2. But IMO such information shouldn't be shown on the very first page of the documentation.

This is an initial page that everyone (I hope ;P) who's new to fuse-box reads. Initial pages are usually tutorials-like, so people tend to copy/paste information without giving too much attention to what they're doing (because usually, they don't really understand what they're doing).

This happened also to me, I've just copy-pasted this line and entered it as a very first line of my new, empty `fuse.js` file. Then, when I reached the [Getting started](https://fuse-box.org/page/getting-started) page, I realized that I did something wrong :)

---
TL:DR
Write about node.js < 8.2 gotchas but somewhere else than on the initial page.
